example/lua/datadog/dd-config.json

```diff
@@ -0,0 +1,4 @@
+{
+  "agent_host": "dd-agent",
```

Comment:
Suggest adding 
"operation_name_override": "nginx.lua_filth"

Since I bet users will copy+paste this, and this will guard against multiple root span names causing issues.

Reply:
Added the override.